### ISSUE TITLE
Range request not supported

### ISSUE THREAD

**Issue by masini** [quarkusio/quarkus#9990]
**Describe the bug**
Range request doesn't work once packaged

**Expected behavior**
While in dev mode everything works fine thanks to FileResource, the implementation of CachedResource used once packaged lack the implementation of the method:

```Java
@Override
public void serveRangeBlocking(OutputStream outputStream, HttpServerExchange exchange, long start, long end) throws IOException {

}

```
**Actual behavior**
The socket will stay there forever untile browser timeout.

**To Reproduce**
1. Download the attached file [beep.ogg.zip](https://github.com/quarkusio/quarkus/files/4776252/beep.ogg.zip)
2. Create a Quarkus project and put the extracted beep.ogg in META-INF/resources
3. `mvn install`
4. Launche the created jar with `java -jar`
5. `curl -v http://localhost:8080/beep.ogg -o /dev/null -H "Range: bytes=0-23606"`

**Environment (please complete the following information):**
 - Quarkus version or git rev: 1.5.0

**Comment by famod** [comment]
It seems this landed in 1.6.0.CR1 via #10220.

**Comment by gsmet** [comment]
Right, thanks!
